mungegithub/submit-queue/deployment/contrib/configmap.yaml

```diff
@@ -3,41 +3,42 @@ kind: ConfigMap
 metadata:
   name: contrib-sq-config
 data:
-  # basic config options.
-  config.http-cache-dir: /cache/httpcache
-  config.organization: kubernetes
-  config.project: contrib
-  config.pr-mungers: blunderbuss,lgtm-after-commit,submit-queue,check-labels
-  config.state: open
-  config.token-file: /etc/secret-volume/token
-  gitrepo.repo-dir: /gitrepos
-  gitrepo.enable-md-yaml: "false"
+  munge-config: |
+    # basic config options.
+    http-cache-dir: /cache/httpcache
```

Comment:
Should we leave things like which, which are defined in the deployment template, as command line flags? So both defining the location of the volume mounts and telling the app where those volume mounts are can be changed together? (This is true for a couple of these flags)